engine/cdn/cdn_admin.go

```diff
@@ -0,0 +1,30 @@
+package cdn
+
+import (
+	"github.com/ovh/cds/engine/api"
+	"github.com/ovh/cds/engine/service"
+)
+
+func (s *Service) getAdminDatabaseSignatureResume() service.Handler {
+	return api.AdminDatabaseSignatureResume(s.mustDB, s.Mapper)
```

Comment:
Maybe you can move this func to a package in engine to avoid dependency from cdn to api, there is for example an existing database pkg to handle shared funcs for database utilities.

Reply:
done